src/test/java/com/fasterxml/jackson/databind/util/BufferRecyclersDatabindTest.java

```diff
@@ -92,14 +100,18 @@ public void testGeneratorWithDequeuPool() throws Exception {
 
     public void testGeneratorWithLockFreePool() throws Exception {
         _testGenerator(JsonRecyclerPools.newLockFreePool());
-        _testGenerator(JsonRecyclerPools.sharedLockFreePool());
+//        _testGenerator(JsonRecyclerPools.sharedLockFreePool());
```

Comment:
Is this intentional?

Reply:
No, sorry. I commented out temporarily to investigate that test and then forgot to uncomment.

Reply:
Makes sense, just wanted to double-check.